src/Action/Server.hs

```diff
@@ -30,7 +33,7 @@ import System.Time.Extra
 import Data.Time.Clock
 import Data.Time.Calendar
 import System.IO.Unsafe
-import Numeric.Extra
+import Numeric.Extra hiding (log)
```

Comment:
These look like they are only heard to satisfy shadow warnings, which seems unnnecessary, as shadowing is not a bad thing.

Reply:
I find the scoping warning very helpful to notice accidental bugs, but ofc it’s a bit of menial work.